sds_data_manager/lambda_code/SDSCode/rds_utils/db_connection.py

```diff
@@ -0,0 +1,115 @@
+import json
+import logging
+import sys
+
+import boto3
+import psycopg2
+from psycopg2 import Error
+
+from .db_ingest_query import DbIngestQuery
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+
+
+class DbConnection:
+    """
+    Class to represent the connection to the RDS database.
+
+    ...
```

Comment:
I see that this is in the `numpydoc` convention, but I don't understand its purpose 🤔